operations/operations/publishing/publishing-guidelines/submitting-documentation-with-your-PR.md

```diff
@@ -12,27 +12,35 @@ You can contribute documentation in one of the following ways:
 
 We don't expect a huge body of documentation or that it's perfectly-written - but rather a clear, concise outline of the change which can be added to our documentation. The content can be provided as a list, rough notes, or you can use the example below for content and structural guidance if your documentation is quite detailed. 
 
-\## Introduction and context\
+----------------------------
+
+\## Introduction and context
+
```

Comment:
We don't include a space between the heading and subtext for other headings. Wasn't sure which was the expected.

Reply:
Oh that is definitely not correct, thanks!